cmd/server/main.go

```diff
@@ -53,28 +56,29 @@ func main() {
 	}
 
 	log.Root().SetHandler(log.LvlFilterHandler(logLevel, log.StreamHandler(os.Stderr, logFormat)))
+	logger := log.New(log.Ctx{"service": *serviceName})
```

Comment:
maybe only do this if serviceName is not empty? perhaps some people don't want it, then could disable it with using an empty serviceName 🤔

Reply:
we have already set the default value as rpc-endpoint. Not sure this will be interesting one for the users